dask_kubernetes/operator/deployment/helm/dask-kubernetes-operator/values.yaml

```
@@ -54,6 +54,9 @@ affinity: {}  # Affinity
 
 priorityClassName: null  # Priority class
 
+livenessProbe:
+  timeoutSeconds: 1
```

@jacobtomlinson I would probably say bump the default but no strong preference
```suggestion
  timeoutSeconds: 5
```